### DISCUSSION TITLE
Configuring multi-user applications with the launcher.

### DISCUSSION THREAD

**Discussion by chaoyangxiao** [Kitware/trame#383] [General]
Hello，
    I looked at the example in trame about configuring a multi-user application using the launcher. When configuring host == "localhost", both the example and my own application work fine, however, when I want to make the application accessible to users on my LAN, I set host == "192.***** " (my local IP address) and the program doesn't work properly and displays a 404notfound error at the URL. I don't know why.
    Looks for your reply.Thanks a lot.
![image](https://github.com/Kitware/trame/assets/111733805/acb466eb-d85a-409f-8c9a-532f3068ce8a)
![image](https://github.com/Kitware/trame/assets/111733805/31399a5f-de8c-4aab-8eed-d8b040a6da84)

**Comment by jourdain** [comment]
The cmd section should be `["python", "./cone.py", "--host", "0.0.0.0", "--port", "$port", "--server", "--authKey", "${secret}"]`

**Comment by jourdain** [comment]
Possibly you should use the `sessionURL: "ws://192.YOUR_IP:${port}/ws"` while all the thing you changed could remain at "localhost"

**Reply by jourdain** [reply]
or `0.0.0.0` for the root `host:`
